.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: C
+dist: xenial
```

Comment:
Is that a real thing now? I thought travis just "upgraded" defaults to 14.04 but only supported precise and trusty.